### ISSUE TITLE
Migrate system and facility health services for VISN 2, 6, 16, 21, 23

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#1946]
This is blocked by the A-Z health services transformation work being done by VAMC upgrade team.

**Comment by kevwalsh** [comment]
Closing this in favour of #2896 and #2897
